app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
+    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_700">#FF3700B3</color>
+    <color name="teal_200">#FF03DAC5</color>
+    <color name="teal_700">#FF018786</color>
+    <color name="black">#FF000000</color>
+    <color name="white">#FFFFFFFF</color>
+    <color name="pomegranate_button_main_activity_color">#90F44336</color>
+    <color name="burntSienna_title_main_activity_color">#EA5853</color>
```

Comment:
please don't mix camel case and snake case

Reply:
https://jeroenmols.com/blog/2016/03/07/resourcenaming/
please use the recommended id convention

Reply:
Fixed!